src/main/kotlin/com/terraformation/backend/nursery/model/BatchModel.kt

```diff
@@ -92,8 +93,10 @@ data class ExistingBatchModel(
       row: BatchesRow,
       subLocationIds: Set<SubLocationId> = emptySet(),
       totalWithdrawn: Int,
+      accessionNumber: String? = null
```

Comment:
A trailing comma would be good here so this line doesn't need to change if we add another argument.